client/src/main/java/model/Sea.java

```diff
@@ -16,31 +15,30 @@
  */
 public class Sea {
 
-  private final List<SeaField> wholeSea;
+  private final List<ClickableField> seaFields;
 
   public Sea() {
-    this.wholeSea = new ArrayList<>();
+    this.seaFields = new ArrayList<>();
   }
 
-  public void addSeaField(SeaField seaField) {
-    wholeSea.add(seaField);
+  public void addSeaField(ClickableField seaField) {
+    seaFields.add(seaField);
   }
 
-  public SeaField getSeaFieldByPosition(Position position) {
+  public ClickableField getSeaFieldByPosition(Position position) {
 
-    Optional<SeaField> field = wholeSea.stream()
+    Optional<ClickableField> field = seaFields.stream()
         .filter(s -> s.position().equals(position))
         .findFirst();
 
     return field.orElseThrow(IndexOutOfBoundsException::new);
   }
 
   public void clearSea() {
```

Comment:
the method name is a little confusing to me. Please add java docs.

Reply:
x